packages/ui/src/pages/VariantEntity/Pathogenicity/GeneTable/conditions/OmimConditionCell.tsx

```diff
@@ -21,7 +21,7 @@ const OmimConditionCell: React.FC<OwnProps> = ({ conditions }) => (
 
                 return (
                     <StackLayout key={index}>
-                        <Text>{geneOmimName}</Text>&nbsp;(OMIM:
+                        <Text>{geneOmimName}</Text>&nbsp;(MIM:
```

Comment:
OMIM to MIM ?

Reply:
Finalement Karine a dit que c'était une erreur de penser que c'était une typo dans cqdg, le texte voulu est bel et bien 'MIM'